framework/src/wifi_manager/wifi_manager.c

```diff
@@ -88,6 +89,8 @@ enum _wifimgr_usr_cb_type {
 	CB_STA_JOINED,
 	CB_STA_LEFT,
 	CB_SCAN_DONE,
+	CB_SOFTAP_DONE, /* This callback does not exist, but is used for stats management. */
+	CB_MAX,
```

Comment:
In case a new callback is included, then we need to change the definition. Here, I want to describe CB_MAX as the maximum number of callback (always), which might be used for other implementation. (For example, an API may want to know the maximum number of callback suppored by wifi_manager.) Thanks for the comment.